media-video/pipewire/pipewire-0.3.24.ebuild

```diff
@@ -0,0 +1,228 @@
+		-Dudev=enabled
+#		-Dudevrulesdir="" # Directory for udev rules (defaults to /lib/udev/rules.d)
+#		-Dsystemd-user-unit-dir="" # Directory for user systemd units (defaults to pkg-config systemd --variable=systemduserunitpath ; prefix awareness expected but not verified)
+		$(meson_feature debug sdl2) # Enables SDL2 dependent code (currently only examples when -Dinstalled_tests=enabled)
+		$(meson_feature extra sndfile) # Enables libsndfile dependent code (currently only pw-cat)
+		# Gentoo added options
+	)
+	meson_src_configure
+}
+
+src_compile() {
+	meson_src_compile
+
+	einfo "Generating ${limitsdfile}"
+	cat > ${limitsdfile} <<- EOF || die
+		# Start of ${limitsdfile} from ${P}
+
+		1000:60000	-	memlock 256
+
+		# End of ${limitsdfile} from ${P}
+	EOF
+}
+
+src_install() {
+	meson_src_install
+
+	insinto /etc/security/limits.d/
+	doins ${limitsdfile}
+
+	if use pipewire-alsa; then
+		keepdir /etc/alsa/conf.d
+
+		dosym8 -r /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d/50-pipewire.conf
+		dosym8 -r /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d/99-pipewire-default.conf
+	fi
+}
+
+pkg_postinst() {
+	elog "It's recommended to install and enable sys-auth/rtkit daemon to allow"
+	elog "PipeWire to obtain restricted realtime capabilities at runtime."
+	elog
+		if use bluetooth; then
+		elog "For oFono support install and enable net-misc/ofono daemon, and"
+		elog "pipewire-media-session should automatically contact it at runtime."
+		elog
+	fi
+	if !use pipewire-alsa; then
+		elog "PulseAudio's ALSA plugin is being used to integrate ALSA applications"
+		elog "into PipeWire's graph. This is fine and will not cause problems but"
+		elog "users should keep in mind that the normally unset PULSE_LATENCY_MSEC"
+		elog "environment variable affects its behaviour. And if the value is too"
+		elog "low, it will cause buffer underruns that will likely result in"
+		elog "PipeWire not playing any audio due to the client buffer never getting"
+		elog "filled on time. Best not to set it at all, contrary to what some"
+		elog "online sources may suggest."
+		elog
+	fi
+	if use pipewire-jack; then
+		elog "JACK support is incomplete and not all programs will work."
+		elog "PipeWire's alternative libraries have been installed to a"
+		elog "non-default location; to use them, prepend pw-jack <application>"
+		elog "before every JACK application invocation."
+		elog
+	fi
+	if use systemd; then
+		elog "Per Gentoo policy installed systemd units must be manually enabled:"
+		elog "systemctl --user disable pulseaudio.service pulseaudio.socket"
+		elog "systemctl --user enable pipewire.socket pipewire-pulse.socket"
+		elog "systemctl --user enable pipewire-media-session.service"
+		elog
+		elog "Rebooting is strongly recommended to avoid surprises from"
+		elog "remnant PulseAudio daemon auto-spawning and surviving logouts."
+		elog
+	else
+		elog "This ebuild tries to do right by OpenRC users by auto-enabling"
```

Comment:
Again, no one understands this construction, even with a dictionary at hand. Suggestion: This ebuild tries do be helpfull to/for (not sure which preposition is correct here) OpenRc users by

Reply:
A Gentoo maintainer already said it's fine in this same discussion on previous ebuild. Besides even if someone does not understand the idiom, it should not really affect the following sentence with clear instructions on how to proceed.